### ISSUE TITLE
NewClientWithPolicyAndHost fails on AWS only

### ISSUE THREAD

**Issue by clanstyles** [aerospike/aerospike-client-go#89]
```
func Aerospike() echo.MiddlewareFunc {
	hosts := []*as.Host{
		as.NewHost("xxxx", 3000),
		as.NewHost("xxxx", 3000),
	}

	client, err := as.NewClientWithPolicyAndHost(nil, hosts...)
	if err != nil {
		panic(err)
	}

	return func(h echo.HandlerFunc) echo.HandlerFunc {
		return func(c *echo.Context) error {
			c.Set(DbKey, client)
			return h(c)
		}
	}
}
```

This code when running on AWS always fails to sync. It doesn't fail locally. If I do a single host connect though given the same NewHost it works fine.

**Comment by Abramovic** [comment]
Have you checked your security groups? You'll need to make sure that ports 3000-3004 are open. 

When we saw this problem what fixed it for us was allowing our Aerospike security group have inbound access to ports 3000-3004 in the same Aerospike security group.

**Comment by ClanStyles** [comment]
Yeah it's not that. 

I have one security group that allows anything for testing.

**Comment by khaf** [comment]
This most likely was problem with default timeout and the way errors were handled. Should be fixed with v1.11.
